test/test-_loadInIframe.js

```diff
@@ -126,6 +126,16 @@ buster.testCase('_loadInIframe',
       assert.equals(this.testObject.window, this._stubs.window);
       assert.equals(this.testObject.document, this._stubs.document);
       assert.equals(this.testObject.$, this._stubs.$);
+    },
+
+    'Throws an exception if $ is not defined on the iframe': function()
+    {
+      assert.exception(function()
+      {
+        this._stubs.$ = undefined;
+        this._stubs.$.withArgs(this._stubs.document).returns(undefined);
```

Comment:
Ah, sorry for the lack of explanation.
While writing a test, I run into a case where I was trying to load a url that I haven't added to the mapped paths yet.

So the problem was caused by the url I was using, but the error I was seeing was coming from jQuery not being attached to the iframe's window object, which was kind of misleading to me.
To help the developer trace back the root cause of the problem, I thought an exception with a proper error message would be helpful in this case.